plutus-core/src/Language/PlutusCore/Evaluation/Machine/ExBudgeting.hs

```diff
@@ -190,7 +191,7 @@ instance ( PrettyDefaultBy config Integer, PrettyBy config exBudgetCat
         , "}"
         ]
 
-newtype ExTally exBudgetCat = ExTally (MonoidalHashMap exBudgetCat ExBudget)
+newtype ExTally exBudgetCat = ExTally (MonoidalHashMap exBudgetCat (Vector ExBudget))
```

Comment:
I'm not sure what the significance of this is, tbh. Maybe this type needs a comment explaining how accumulation works?

Reply:
Gives you a more detailed view on how much you spent, so you can see the individual numbers. Next step would be to attach some more information (e.g. call stack, line number) to each spot.